docs/api-docs/storefront/graphql/graphql-site-content.mdx

```diff
@@ -0,0 +1,1004 @@
+                altText
+                isDefault
+              }
+              seo {
+                pageTitle
+                metaKeywords
+                metaDescription
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+<Tab>
+
+```json filename="Example query: Get blogs" showLineNumbers copy
+{
+  "data": {
+    "site": {
+      "content": {
+        "blog": {
+          "name": "Blog",
+          "description": "",
+          "path": "/blog/",
+          "posts": {
+            "edges": [
+              {
+                "node": {
+                  "entityId": 4,
+                  "name": "How to seed your garden",
+                  "tags": [
+                    "Best Reviews",
+                    "Most Popular"
+                  ],
+                  "htmlBody": "<p>1. Sow at a suitable depth<br>2. Plant at the specified spacing for your plant</p>",
+                  "plainTextSummary": "1. Sow at a suitable depth2. Plant at the specified spacing for your plant",
+                  "author": "Anonymous",
+                  "path": "/blog/how-to-seed-your-garden/",
+                  "publishedDate": {
+                    "utc": "2023-06-06T19:29:58Z"
+                  },
+                  "thumbnailImage": {
+                    "url": "https://cdn11.bigcommerce.com/s-o8ertuc7vw/images/stencil/100x200/uploaded_images/flower.jpg",
+                    "urlOriginal": "https://cdn11.bigcommerce.com/s-o8ertuc7vw/images/stencil/original/uploaded_images/flower.jpg",
+                    "altText": "How to seed your garden",
+                    "isDefault": true
+                  },
+                  "seo": {
+                    "pageTitle": "How to seed your garden",
+                    "metaKeywords": "Best Reviews,Most Popular",
+                    "metaDescription": "Flower products"
+                  }
+                }
+              },
+              {
+                "node": {
+                  "entityId": 1,
+                  "name": "Indoor Planting",
+                  "tags": [
+                    "Holiday",
+                    "Lowest Price"
+                  ],
+                  "htmlBody": "<p>1. Pay attention to humidity<br>2. Provide adequate lighting</p>",
+                  "plainTextSummary": "1. Pay attention to humidity2. Provide adequate lighting",
+                  "author": "Anonymous",
+                  "path": "/blog/indoor-planting/",
+                  "publishedDate": {
+                    "utc": "2014-02-15T19:31:55Z"
+                  },
+                  "thumbnailImage": null,
+                  "seo": {
+                    "pageTitle": "Indoor Planting",
+                    "metaKeywords": "Holiday,Lowest Price",
+                    "metaDescription": ""
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+</Tab>
+</Tabs>
+
+### Rendered regions
```

Comment:
Since the Blog templates in Cornerstone don't contain any regions by default, I'm wondering if we should leave this section out. Will it just confuse people?
https://github.com/bigcommerce/cornerstone/blob/master/templates/pages/blog-post.html

Reply:
✔️  Thanks for mentioning. If it's not default behavior, it might confuse people, so I took it out.